gui/wxpython/core/debug.py

```diff
@@ -48,8 +48,9 @@ def SetLevel(self):
             self.debuglevel = 0
             sys.stderr.write(
                 _(
-                    "WARNING: Ignoring unsupported wx debug level (must be >=0 and <=5). {0}\n"
-                ).format(e)
+                    "WARNING: Ignoring unsupported wx debug level "
+                    f"(must be >=0 and <=5). {e}\n"
+                )
```

Comment:
This string is translatable, it has the `_(...)` call around it (the function to translate from English to the locale language is named `_`). You should not be getting warning from anything to convert this to an f-string because the underscore function is called on that string first an only then the string is formatted (`{e}` is substituted).

Splitting the line and replacing the number-based formatting would be okay otherwise. Given the context, just keep `{0}` there and only split the line.

Reply:
You are right. It was unnecessary so changed back to original and adopted E501 fix. As long as I remember, I did only once like this.
```
sys.stderr.write(
    _(
        "WARNING: Ignoring unsupported wx debug level (must be >=0 and "
        "<=5). {0}\n".format(e)
    )
)
```